extensions/amp-story/1.0/amp-story-page-attachment.css

```diff
@@ -172,7 +172,7 @@ amp-story[desktop] .i-amphtml-amp-story-page-attachment-ui-v2.i-amphtml-story-pa
   background-color: var(--i-amphtml-outlink-cta-text-color) !important;
   opacity: .6 !important;
   transform-origin: left !important;
-  animation: progress-bar-animation both 1s !important;
+  animation: progress-bar-animation .6s both cubic-bezier(0.4, 0.0, 1, 1) !important;
```

Comment:
Why is this animation .6s but the delay is 500ms?

Reply:
Added a comment about that above the const. Do you think it would be good to add it here in the CSS file also?
https://github.com/ampproject/amphtml/pull/34477/files#diff-16b74d4df032ffe495097ffaa109e047f5e1291b5e91adfebd4d6ed8e26b69d3R57-R60

Reply:
Oh my bad, didn't read the comment. No need for extra documentation